app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -229,12 +226,19 @@ function ModelConfigDialogContent(props: ModelConfigDialogContentProps) {
 
   const query = useLazyLoadQuery<ModelConfigButtonDialogQuery>(
     graphql`
-      query ModelConfigButtonDialogQuery($providerKey: GenerativeProviderKey!) {
+      query ModelConfigButtonDialogQuery(
+        $providerKey: GenerativeProviderKey!
+        $modelName: String
+      ) {
         ...ModelProviderPickerFragment
-        ...ModelPickerFragment @arguments(providerKey: $providerKey)
+        ...ModelPickerFragment
```

Comment:
why does the model picker fragment need teh model name now? seems a bit odd

Reply:
because parameters can vary by model name, not just provider

Reply:
for example, over long enough time different models will be accessed using different APIs, the invocation parameters supported by those APIs are not necessarily the same (e.g. o1 not supporting temperature or other weighting parameters)

Reply:
but the model picker is just fetching models by provider, not invocation params

Reply:
oh this is for the model picker, yeah it isn't using it

Reply:
I was getting errors from the server until I provided this. Let me see if it will let me remove it again

Reply:
Yeah it works without it now, must have been interstitial weirdness that we've now fixed